### DISCUSSION TITLE
React Native Browser Environment Variables

### DISCUSSION THREAD

**Discussion by SamratSahoo** [vercel/community#785] [Help] [answered]
I am trying to use an environment variable with my react native application but when deploying to Vercel, the environment variable was not exposed to the browser (I have added the env variable to the project settings). 

I know with Next.js you can [expose an env variable by prefixing with NEXT](https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser) -- I was wondering if there was something similar I could do with React Native to use the environment variable in the browser? 

Thanks in advance!

**Comment by thomas-corrado** [comment]
My environment variables are also not working (particularly using a Google Sheets API), and I can't figure out why!

**Reply by amyegan** [reply]
Hi @thomas-corrado! It looks like your question is a little different from this discussion topic. Can you [create a new discussion](https://github.com/vercel/community/discussions/new) with details about your app, Project Settings, and your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example)? That will help me investigate what is going wrong for you.

**Comment by amyegan** [comment, marked as answer]
Hi @SamratSahoo! You might be able to do something similar with React Native, but I don't know enough about it to be able to say for sure. I _think_ it uses the same `REACT_APP_` prefix as React apps. I would recommend asking the [React Native community](https://reactnative.dev/help) since they will know more about library specifics.
